mining.md

```diff
@@ -270,6 +270,31 @@ An eligible miner broadcasts the completed block to the network (via [block prop
 
 ### Block Rewards
 
+Over the entire lifetime of the protocol, 1,400,000,000 FIL (`TotalIssuance`) will be given out to miners. These funds are initially held by the network account actor, and are transferred to miners in blocks that they mine. The reward amount remains fixed for a period of 1 week (20,160 blocks, the `AdjustmentPeriod`) and is then adjusted.
```

Comment:
1) The 20,160 assumes 30 sec block time. May be worth mentioning this.

2) Moreover, Unlike with BTC where difficulty is tuned for the block time to remain constant on expectation, we do not do this (yet). Should we leave a TODO here regarding difficulty adjustments pertaining to VDFs? If so, I feel comfortable having the period be counted in time, else we should probably have it be a number of blocks (this would not be as good imo).

3) Why 1 week? I haven't thought this through much, but default value that comes to mind is 2 for obvious reasons. I get why you would want to shorten this, any reason to make it longer?

Reply:
I have no strong reasoning one way or the other on why 1 week. Its somewhere between 'often enough' and 'not too often'. And its long enough that any variance from EC should get averaged out pretty well. 

Thoughts?

Reply:
Yeah I think that's fine. Might have said 2 weeks since that's what BTC does, but we can adjust in testing as needed. It seems like a good starting point.